90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -100,6 +100,16 @@ mount_zfs() {
   return ${ret}
 }
 
+# prints: first sort key
+# returns: nothing
+
+get_sort_key() {
+  local sort_key
+  sort_key="${zbm_sort%%;*}"
+  zdebug "Using sorting key ${sort_key}"
+  echo -n "${sort_key}"
```

Comment:
Maybe `echo -n "${sort_key:-name}"` to prevent degenerate output.

Reply:
Good call, done.